syntax/hoon.vim

```diff
@@ -6,20 +6,28 @@ if exists("b:current_syntax")
 endif
 
 set autoindent
-map g/ /+\(+\\|=\\|\-\)  
+map g/ /+\(+\\|=\\|\-\\|\$\\|\*\)  
 nmap gs :let varname = '\<<C-R><C-W>\>'<CR>?++  <C-R>=varname<CR><CR>
 set tabstop=2
 set expandtab
 set shiftwidth=2
-" nmap gc :let &colorcolumn=join(range(81,999),",")<CR>
-" nmap gC :let &colorcolumn=join(range(999,999),",")<CR>
-" nmap ge :vertical resize 85<CR>
+
+" Allow gq, o, and <Enter> to understand comments
+let b:comment_leader = '::  '
+set comments=:::
+set fo+=rq
+
+" Don't wrap code at 72 characters, just comments
+set fo-=t
+
+" limit to 72 characters
+set tw=72
```

Comment:
Why limit to 72 characters, not 80?

Reply:
80 character comments are generally bad, because they're very sensitive to, eg indenting one level or commenting out a whole section of code including comments.  Personally, I also keep my panes slightly narrower than 80 characters, so it's a pain if your comments hit the limit.

Reply:
Hmm, I think our current standard is 80 characters. Certainly Ford Turbo has a lot of 80-character comments.